client/src/components/NavBar/AuthBars/Notifications/Notification/useStyles.ts

```diff
@@ -0,0 +1,57 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
+  notification: {
+    display: 'grid',
+    gridTemplateColumns: '25fr 75fr',
```

Comment:
Rather than using CSS to apply a grid, you can use MUI Grid container/items to do the same :)

Ref: https://mui.com/components/grid/